src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtHttpCall.java

```diff
@@ -20,7 +20,7 @@ public class ExtHttpCall {
 
     String responsebody;
 
-    BidderCallType calltype;
+    BidderCallType callType;
```

Comment:
Stop. You should never change production model for tests. And you should now use production model for tests too. Create a separate model for test.

Reply:
I did a typo there. I meant that you should not to touch production model in general.

Still, you will need to undo your changes to the prod code.